src/transformers/models/flaubert/modeling_flaubert.py

```diff
@@ -448,7 +446,6 @@ def __init__(self, config):  # , dico, is_encoder, with_output):
 
         # Initialize weights and apply final processing
         self.post_init()
-        self.register_buffer("position_ids", torch.arange(config.max_position_embeddings).expand((1, -1)))
```

Comment:
Does it still work with the removal of this buffer?

Reply:
Probably not, good catch :sweat_smile:

Reply:
Ah it was a duplicate (not shown by the diff). You can scroll to line 452 below to see it again defined with `persistent=False`.